### DISCUSSION TITLE
“Load cannot follow more than 20 redirects / Too many redirects” after enable Cloudflare CDN

### DISCUSSION THREAD

**Discussion by FLYEMOJ1** [mastodon/mastodon#22197] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/53422421/206831011-2b46b0f0-aaef-43fb-86e3-f9b680d2179d.png)

followed by Mastodon Ubuntu Deploy Guide (sorry for the bad english ;w;)

**Comment by c04m3** [comment, marked as answer]
Check your SSL settings to see if there are any abnormalities, as this could be the cause of the error occurring.
檢查您的 SSL 設定，看看有沒有異常，這可能是造成此錯誤發生的原因。

若您不擅長使用英文，其實可以在中文區的大型站台直接貼文發問，會更快得到答案。

**Reply by FLYEMOJ1** [reply]
感谢您的答复。
本问题已解决，
因Cloudflare SSL未设置到严格模式而导致的持续重定向。
麻烦您了

**Reply by iRoySwift** [reply]
我遇到同意的问题，也是cloudflare ssl未设置严格模式
